docs/documentation/js/release-info.js

```diff
@@ -138,10 +147,28 @@ document.addEventListener("DOMContentLoaded", function () {
         state.innerText = ` at ${date.format("dddd, mmmm dd, yyyy")} at ${date.format("h:MM TT")}`;
 
         state.prepend(link);
-        if (getValueCell(item, 'state') === 'success') {
-          state.prepend("✅ ");
-        } else {
-          state.prepend("❌ ");
+        switch (getValueCell(item, 'state')) {
+          case 'inactive':
+            state.prepend(`${icons.inactive} `);
+            break;
+          case 'queued':
+            state.prepend(`${icons.queued} `);
+            break;
+          case 'in_progress':
+            state.prepend(`${icons.in_progress} `);
+            break;
+          case 'pending':
+            state.prepend(`${icons.pending} `);
+            break;
+          case 'success':
+            state.prepend(`${icons.success} `);
+            break;
+          case 'failure':
+            state.prepend(`${icons.failure} `);
+            break;
+          case 'error':
+            state.prepend(`${icons.error} `);
+            break;
```

Comment:
So, we map icons to statuses with `icons` map and then again map statuses to icons with switch/case. We map this twice.

I'd do it differently, but since it will work, the change is up to you :)

```js
// Instead of switch/case, one can write this 
const icon = icons[item.state]
if (icon) {
  state.prepend(`${icon} `)
}

```